src/translations/it/translations.ts

```diff
@@ -312,6 +312,14 @@ export const TRANSLATIONS_IT = {
       title: "L'operazione è stata presa in carico",
       body: "Riceverai l’esito del pagamento all'indirizzo {{useremail}}",
     },
+    survey: {
+      title: "Puoi dirci com'è andata?",
+      body: "Raccontaci la tua esperienza con il pagamento e aiutaci a migliorare.",
+      link: {
+        text: "Vai al sondaggio",
+        href: "https://pagopa.gov.it/diccilatua/ces-pagamento",
```

Comment:
Add a trail slash... it's more safe
`https://pagopa.gov.it/diccilatua/ces-pagamento/` 